README.md

```diff
@@ -61,13 +61,13 @@ See [packages/ia-prototype-apps/README.md](packages/ia-prototype-apps/README.md)
 
 Prototypes can be run like this:
 ```
-cd packages/prototypes
-yarn run parcel prototypes/examples/example-hello/index.html
+cd packages/ia-prototypes-apps
```

Comment:
we will leave directory name the same, let's update these to reflect directory name.

Reply:
I didn't understand.

Reply:
the directory is `packages/ia-prototype-apps`. please reflect this in your code.

Reply:
I haven't made any change in the directory structure. I only made changes in the README.md file.